crypto/objects/objects.txt

```diff
@@ -1524,7 +1524,7 @@ secg-scheme 14 3 : dhSinglePass-cofactorDH-sha512kdf-scheme
 1 3 6 1 4 1 311 60 2 1 3	: jurisdictionC		: jurisdictionCountryName
 
 # SCRYPT algorithm
-1 3 6 1 4 1 11591 4 11		: id-scrypt
+1 3 6 1 4 1 11591 4 11		: scrypt            : id-scrypt
```

Comment:
Hmmm, still not sure about this. Just tried it like this:

```
1 3 6 1 4 1 11591 4 11          : id-scrypt         : scrypt
```

But then the NID symbol name changes from NID_id_scrypt to NID_scrypt and I get

```
crypto/asn1/p5_scrypt.c: In function ‘pkcs5_scrypt_set’:
crypto/asn1/p5_scrypt.c:202:38: error: ‘NID_id_scrypt’ undeclared (first use in this function)
     keyfunc->algorithm = OBJ_nid2obj(NID_id_scrypt);
```

Is this okay, i.e., should I simply change all occurences of NID_id_scrcypt to NID_scrypt?